.github/workflows/cypress.yml

```diff
@@ -0,0 +1,64 @@
+name: '[QE] Cypress Runner'
+
+on:
+  workflow_dispatch:
+  schedule:
+    # 8:30 AM UTC = 4:30 AM EDT
+    - cron: '30 8 * * *'
```

Comment:
Are you sure to schedule it right now?

Reply:
yes
8 am utc is b2b
So, I used 8:30 am utc for checkout ui custom